### DISCUSSION TITLE
q-header reveal state init

### DISCUSSION THREAD

**Discussion by nseb** [quasarframework/quasar#15576] [Ideas / Proposals]
Hello , 
it's possible to add reveal-state accessible for the init the reveal: 
My case , I have an header with reveal mode , in the header i have tab . I I scroll header hide but tab is always visible ( that's what I want). Now , I a click on another tab , as it is the same root layout, I would like the header to remain non-reveal because otherwise it reappears and the tab bar goes down.
The problem is that I can't fix the starting state of the reveal

Best Regards

**Comment by metalsadman** [comment]
tie the state of it to a store.
